src/extensions/interactive.ts

```diff
@@ -123,7 +123,7 @@ module.exports = (toolbox: CycliToolbox) => {
     }).prompt()
 
     if (isCancel(confirmed)) {
-      throw Error('The script execution has been canceled by the user.')
+      throw CycliError('The script execution has been canceled by the user.')
```

Comment:
Maybe we should also have some predefined errors so we don't repeat this every time? Like const CycliErrorCanceled = new CycliError(...), wdyt?

Reply:
It could be even just here at the top of file as it seems to be just in this file.

Reply:
good idea, done in 8cc3ad70823b34731eae1f07ddab2e262ac195e9